lib/simctl.js

```diff
@@ -236,5 +237,19 @@ async function getDevices (forSdk:string = null):Object {
   return devices;
 }
 
+/**
+ * Gets base64 screenshot for device (xcode >= 8.1 only)
+ * @param {string} udid 
+ */
+async function getScreenshot (udid:string):string {
+  let tempScreenshotDir = path.resolve(__dirname, 'tmp-screenshots');
+  await fs.mkdir(tempScreenshotDir);
```

Comment:
This stuff is all done correctly using https://github.com/appium/appium-support/blob/master/lib/tempdir.js